### DISCUSSION TITLE
Replace downloads badge in README with a more consistent and accurate badge.

### DISCUSSION THREAD

**Discussion by ayushjain01** [sqlalchemy/sqlalchemy#11460] [Ideas]
Current badge in the README displayed the download counts seems to be inconsistent and less accurate since the download counts for sqlalchamy is significantly greater than what the badge shows. 

This badge from pip Trends is more accurate with the numbers and is refreshed daily.
[![sqlalchemy Downloads Last Month](https://assets.piptrends.com/get-last-month-downloads-badge/sqlalchemy.svg 'sqlalchemy Downloads Last Month by pip Trends')](https://piptrends.com/package/sqlalchemy) [![Downloads](https://static.pepy.tech/badge/sqlalchemy/month)](https://pepy.tech/project/sqlalchemy)
I believe this should replace the existing badge. Also, if necessary, the link from the badge to the package's pip Trends page can be removed. We just want to showcase a badge we have created.

**Comment by CaselIT** [comment]
Hi,

I don't have a particular preference one way or the other.
The difference may be because the current badge counts only the 2.0 version?

@zzzeek thoughts?

**Reply by ayushjain01** [reply]
I don't think that's the case, these badges are generally inconsistent and I've verified them with other packages too.

**Comment by zzzeek** [comment]
so we are regularly (every year or so) hit with these requests to change our badges to someone else's badge.   These badges are of course very valuable to their creators since they allow stats usage of packages to be streamed to their personal servers.

Given that, it would be appreciated when you suggest a very specific badge that I've never heard of, you least disclose, if I'm not misunderstanding (please correct me if I'm wrong), that you are [personally involved with that badge project](https://github.com/tankala/learn-pip-trends), as this reveals some of the rationale why this very unusual badge is being recommended.

Overall, all badges receive the exact same information from the pypi and github page they are displayed on, so I dont really see how one badge could be more "accurate" than another, and if there are indeed accuracy problems with our current badge, I'd want to identify how that is actually known and first report it as a bug to our current badge providers which appear to be both shields.io and pepy.tech.    thanks!

**Comment by ayushjain01** [comment]
As you mentioned, I am personally associated with the badge project and have built the badge generation module.

You’re correct that all badges receive the same information, but for some reason, there is a significant difference in the counts shown on the two badges. Our count matches the following query:

```SQL
SELECT COUNT(*) AS num_downloads
FROM `bigquery-public-data.pypi.file_downloads`
WHERE file.project = 'sqlalchemy'
  -- Only query the last 30 days of history
  AND DATE(timestamp)
    BETWEEN DATE_SUB(CURRENT_DATE(), INTERVAL 30 DAY)
    AND CURRENT_DATE()
```
We’re not quite sure why the current badge has inconsistent counts, but it's the same across all packages. You can refer to the [Python Packaging User Guide](https://packaging.python.org/en/latest/guides/analyzing-pypi-package-downloads/#counting-package-downloads) for more information.

Regarding the badge's appearance, we wanted it to stand out to avoid being confused with shields.io, so we chose a completely different design scheme.

Also, I’m not trying to pressure you into using the badge by pip trends; I just thought we could start a discussion since you already have a badge and might as well display the right stats.

**Comment by ixmatus** [comment]
A colleague of this person tried to open this exact PR against a project I'm the maintainer for as well (https://github.com/ixmatus/inflector/pull/16). This is a promotional campaign and they (or colleagues) appear to be doing it on reddit too, I think this is suspicious activity. They are creating an external dependency on a web property that (as far as I can tell) is not officially blessed by pypi.

Users can get package statistics from `libraries.io` which is linked to directly by pypi itself.

**Comment by ayushjain01** [comment]
I have clearly mentioned why this badge is better than `libraries.io` and other badges out there due to the significant difference in the actual counts and the counts displayed by these badges. Our badges are accurate up to the last digit and this is just to share what we have built and if needed the link to the pip trends page can also be removed.
